volttrontesting/platform/auth_tests/test_auth_control.py

```diff
@@ -274,16 +274,16 @@ def test_auth_remove(auth_instance):
     entries = auth_list_json(platform)
     assert len(entries) > 0
     i = 0
-    while len(entries) < len_entries and i < auth_retry:
-        gevent.sleep(1)
+    while len(entries) <= len_entries and i < auth_retry:
+        gevent.sleep(i)
         entries = auth_list_json(platform)
         i += 1
     auth_add(platform, _auth_entry6)
     entries = auth_list_json(platform)
     assert len(entries) > 0
     i = 0
-    while len(entries) < (len_entries + 1) and i < auth_retry:
-        gevent.sleep(1)
+    while len(entries) <= (len_entries + 1) and i < auth_retry:
```

Comment:
why is this one len_entries + 1 while the others are len_entries?
why are we changing these?

Reply:
because there are 2 auth_add in the test case at this point and len_entries is length before any add by test case